.spellcheckwordlist.txt

```diff
@@ -434,6 +434,7 @@ not
 npm
 nssm
 nvm
+octicons
```

Comment:
Is that a word?

Reply:
Well, no. But the spell checker doesn’t like the `:octicons-*` convention and [it’s a real thing](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/?h=icons).